src/Spectre.Service/Dependencies/MockFilesSystemModule.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Abstractions;
+using System.IO.Abstractions.TestingHelpers;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Ninject.Modules;
+
+namespace Spectre.Service.Dependencies
+{
+    /// <summary>
+    /// To do
```

Comment:
Fix all documentation comments.

Reply:
Documentation will be added if some final solution is found,